tasks/get-ci-results.yml

```diff
@@ -20,7 +20,7 @@ spec:
         #! /usr/bin/env bash
         PREFLIGHT_RESULTS_EXISTS="$(params.preflight_results_exists)"
         if [ $PREFLIGHT_RESULTS_EXISTS == "true" ]; then
-          echo "Preflight results already exists- skipping"
+          echo -n "Preflight results already exists- skipping"
```

Comment:
`-n` option is useful only when returning a value as a result. For classic log messages, the option doesn't have to be there.

Reply:
Ah, yes. Sorry, `find and replace` went out of control- thanks for your catch!